tests/integration/utils.py

```diff
@@ -248,7 +248,7 @@ def check_url(self, link_url, endpoint, query_params=None):
 
 # Check response of an endpoint for response code, schema, self link
 def test_endpoint(self, endpoint, resource, response_code, query_params=None,
-                  nullable_fields=[]):
+                  nullable_fields=None):
```

Comment:
So, if the `nullable_fields` became `None` by default and `check_schema` was called at [line 256](https://github.com/osu-mist/express-api-skeleton/pull/17/files/8df2546a986eeafb6cd2a28fabdd7eee9a2daea1..aee5294b0bd108ff8844ffbb0a7a663c705abd52#diff-38d98ae9c99a4a24f98581f0ba32b02aR256), then we will eventually hit the [line 193](https://github.com/osu-mist/express-api-skeleton/pull/17/files/8df2546a986eeafb6cd2a28fabdd7eee9a2daea1..aee5294b0bd108ff8844ffbb0a7a663c705abd52#diff-38d98ae9c99a4a24f98581f0ba32b02aR193) and cause an error that `TypeError: argument of type 'NoneType' is not iterable`